README.md

```diff
@@ -7,3 +7,53 @@ The challenges are inspired by the [100 days of code project](https://www.100day
 
 [Телеграм-бот](https://t.me/days_of_code_bot) для проведения марафонов "N days of code" в телеграм-чатах.  
 Марафоны по мотивам проекта [100 дней кода](https://www.100daysofcode.com/)  
+
+## Contributing
+
+We are using [GitHub Issues](https://github.com/lightalloy/days-of-code-bot/issues) for our public bugs. If somebody claims an issue but doesn’t follow up for more than two weeks, it’s fine to take it over but you should still leave a comment.
+
+If you send a pull request, do it against the master branch. For our branches we are using a scheme of (github issue key)-(optional desc/keyword), .e.g. ```34-blah_blah```. We also use this scheme at the beginning of commit message.
```

Comment:
I think in this case desc/keyword in the branch name is not optional: the branch name shouldn't be only a number.
+ I would replace "We also use this scheme at the beginning of commit message." with smth like "Please, reference the ticket number in the commit message and in the pull request name and description".